### ISSUE TITLE
Documentation of Expected Outputs

### ISSUE THREAD

**Issue by mjfox3** [YannickJadoul/Parselmouth#23]
I am trying to use features of the parselmouth module and I'm finding that some methods lack enough documentation to know what their output is. Usually I can work backwards from the output of the same function in Praat itself, but in the case of the MFCC(CC).extract_features() method I do not see an equivalent function in Praat. Moreover, the output from this method does not match the output from to_matrix() or other methods for getting values from the object. Using parselmouth in general would be made a lot easier by updating the documentation with expected outputs and what they are.

**Comment by YannickJadoul** [comment]
@mjfox3 Agreed. There's still a bunch of stuff I want to do in Parselmouth, and better docs is one of them. I've started work on this in the version of the docs of the next release, e.g. documenting [`parselmouth.praat.call`](https://parselmouth.readthedocs.io/en/latest/api_reference.html#parselmouth.praat.call), but there's a lot of work to do. So any help is welcome :-)

You are correct that most functions map directly onto Praat (which is one of the reasons documentation has not been at the top of my todo-list). `parselmouth.MFCC.extract_features` (and `parselmouth.MFCC.to_matrix_features`, I'm not sure why I made the first alias, anymore) corresponds to the (by default hidden) Praat action `To Matrix (features)`. This seems badly documented in Praat, as well, but in the Praat C/C++ code, there's a comment:

```
/* 1: cepstral difference function (d)
 * 2: spectral stability (dstab)
 * 3: spectral center of gravity (gs)
 * 4: stable internal duration
 */
```
